packages/dds/tree/src/tree/anchorSet.ts

```diff
@@ -146,6 +146,18 @@ export class AnchorSet {
         return parentPath?.tryGetChild(path.parentField, path.parentIndex);
     }
 
+    private deepDelete(nodes: readonly PathNode[]): void {
```

Comment:
a comment on this method (or in it) saying what it does would be good.

```suggestion
    /**
     * Mark all nodes in the subtrees rooted at a node in `nodes` as deleted.
     */
    private deepDelete(nodes: readonly PathNode[]): void {
```